txtl-liposome_thin-film.md

```diff
@@ -0,0 +1,109 @@
+# TX-TL Liposome Using Thin Film Hydration
+
+Version 0.1.0, 14 August 2018
+
+Contributors:
+Aaron Engelhart (University of Minnesota),
+Jan Gregrowicz (Caltech),
+Joseph Heili (University of Minnesota),
+Zoila Jurado (Caltech),
+Neha Kama (Northwestern),
+Akshay Maheshwari (Stanford),
+Richard Murray (Caltech),
+Milena Popovic (Blue Marble Space),
+Pasquale Stano (University of Salento),
+Kazhito Tabata (University of Tokyo),
+Paola Torre (University of Pennsylvania).
+
+## Overview and Materials
+
+This protocol prepares phospholipid films with and without a lissamine
+rhodamine membrane dye to be hydrated later to prepare lipid
+vesicles.  This procedure works best for studies with bulk small
+unilamellar vesicles, but can be used to prepare giant (>1 um)
+vesicles (just skip the extrusion step).
+
+### Materials
+
+A. Glass vials (any will do, but we use [2mL Fisherbrand Class B Clear
+   Glass Threaded vials: cat #:
+   03-339-21A](https://www.fishersci.com/shop/products/fisherbrand-class-b-clear-glass-threaded-vials-with-closures-packaged-separately-12/p-204738)
+
+B. Glass syringes of various sizes (ex. Hamilton gastight cat #:
+   14-815-238, but any will do) for use with lipids and chloroform
+
+C. Phosphate buffered saline 290 mOsm (Sigma, P4417-100TAB)
+
+D. Mini Extruder (Avanti Polar Lipids) 
+
+![Materials for TX-TL liposome protocol](txtl-liposome_thin-film-materials.png)
+
+## Method:
+
+Final reaction conditions:
+
+| Reagent | Source | Final working concentration |
+| ------- | ------ | --------------------------- |
+| DOPC    | Avanti 850375C | 2.5 E -5 mols/vial (1 mL of hydration yields 20 mM vesicles)* |
```

Comment:
Fix formatting for working concentration + fix issue noted above and repeated here.